openssl-dynamic/src/main/java/io/netty/internal/tcnative/SSL.java

```diff
@@ -31,9 +31,40 @@
  */
 package io.netty.internal.tcnative;
 
+import static io.netty.internal.tcnative.NativeStaticallyReferencedJniMethods.sslErrorNone;
```

Comment:
consider reverting these import changes to make merging easier. It doesn't look like these changes are related to this PR.